wagtail/admin/templates/wagtailadmin/pages/history.html

```diff
@@ -4,7 +4,7 @@
 {% block titletag %}{% blocktrans trimmed with title=subtitle %}Page history for {{ subtitle }}{% endblocktrans %}{% endblock %}
 
 {% block actions %}
-    <a href="{% url 'wagtailadmin_pages:edit' page.id %}" class="button bicolor icon icon-edit">{% trans "Edit this page" %}</a>
+    <a href="{% url 'wagtailadmin_pages:edit' page.id %}" class="button bicolor button--icon">{% icon name="edit" wrapped=1 %}{% trans "Edit this page" %}</a>
```

Comment:
Note on `wrapped=1` – not the nicest API but essentially this is needed for `button--icon` buttons. The buttons look bigger like this than with the icon font, which we’ve accepted in other parts of the UI.